docs/config-options-listings.mdx

```diff
@@ -0,0 +1,38 @@
+---
+id: config-options-listings
+title: Listings For All Buildbuddy Options
+sidebar_label: Options Listings
+---
+
+import FreeServerConfig from '../bazel-bin/server/cmd/buildbuddy/yaml_doc/buildbuddy_server_documented_defaults.mdx'
+import EnterpriseServerConfig from '../bazel-bin/enterprise/server/cmd/server/yaml_doc/buildbuddy_enterprise_server_documented_defaults.mdx'
+import ExecutorConfig from '../bazel-bin/enterprise/server/cmd/executor/yaml_doc/buildbuddy_executor_documented_defaults.mdx'
+
+Provided below are working, documented YAML configs for each Buildbuddy binary
+containing every option that that binary accepts, each set to the default value
+for that option. Any option can be specified on the command line by joining the
+names of the nesting structures, if any, and the option name with a `.`
+separating each element.
+
+E.g.,
```

Comment:
nit: I remember reading a technical writing "best practice" somewhere not to use "E.g." since it's not as easily recognized by non-native speakers. It recommended "For example:" as an alternative.

Reply:
```suggestion
For example:
```